dapps/marketplace/src/pages/transaction/Transaction.js

```diff
@@ -150,10 +140,28 @@ const Transaction = props => {
             </>
           )
 
+          const VerticalSeparator = <div className="vertical-separator my-3" />
```

Comment:
Can we move these outside the render method and maybe make then components?

Reply:
Or just use a div with the className vertical-separator?

Reply:
I would use a div with the class name, but didn't want to copy-paste the code 3 times. Horizontal separator also has a nested div. 

My thinking for moving that to variables was to make code less error prone. Since you can quickly change properties of one vertical/horizontal separator and not the other one. When defined with a variable that is not a problem. 

If you think necessary I can use the divs inline the rest of the jsx.